src/js/pouchPersisted.js

```diff
@@ -0,0 +1,253 @@
+/*
+Copyright 2016 OCAD University
+
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one these
+Licenses.
+
+You may obtain a copy of the ECL 2.0 License and BSD License at
+https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.txt
+*/
+
+/* global fluid, floe */
+
+(function ($, fluid) {
+
+    "use strict";
+
+    // A datasource grade using pouchDB via gpii-pouch
+    fluid.defaults("floe.dashboard.dataSource.pouchDB", {
+        gradeNames: ["fluid.dataSource"],
```

Comment:
This should be reorganised to reuse the logic from the CouchDB dataSource grade rather than duplicating it in the set and get methods